### DISCUSSION TITLE
Welcome to neon Discussions!

### DISCUSSION THREAD

**Discussion by kelvich** [neondatabase/neon#1258] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DanDWB** [comment]
I'm Daniel, a Technology enthusiast looking at learning the craft of what I preach.

I wanted to ask the Neon team something in a more private setting. Feel free to reach me 
on Daniel@rekallconsulting.com
